iroh-net/src/dns.rs

```diff
@@ -14,23 +17,83 @@ fn get_resolver() -> Result<TokioAsyncResolver> {
     let (config, mut options) =
         trust_dns_resolver::system_conf::read_system_conf().unwrap_or_default();
     // lookup IPv4 and IPv6 in parallel
-    options.ip_strategy = trust_dns_resolver::config::LookupIpStrategy::Ipv4AndIpv6;
+    options.ip_strategy = trust_dns_resolver::config::LookupIpStrategy::Ipv4thenIpv6;
 
     let resolver = AsyncResolver::tokio(config, options);
     Ok(resolver)
 }
 
+/// Resolve IPv4 and IPv6 in parallel
```

Comment:
This should probably elaborate a little why we don't use the `Ipv4AndIpv6` strategy.

Reply:
added some details